local/src/server/verifier-server.cc

```diff
@@ -330,7 +330,7 @@ swoc::Errata do_listen(swoc::IPEndpoint &server_addr, bool do_tls) {
   int socket_fd = socket(server_addr.family(), SOCK_STREAM, 0);
   if (socket_fd >= 0) {
     // Be agressive in reusing the port
-    int ONE = 1;
+    static constexpr int ONE = 1;
```

Comment:
That seems wrong - you're passing the address of a `constexpr`? That's not even guaranteed to exist. Also, doesn't `setsockopt` modify that argument? How can it do that now?